lib/NodeStuffInWebError.js

```diff
@@ -0,0 +1,33 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Ivan Kopeykin @vankop
+*/
+
+"use strict";
+
+const WebpackError = require("./WebpackError");
+const makeSerializable = require("./util/makeSerializable");
+
+/** @typedef {import("./Dependency").DependencyLocation} DependencyLocation */
+
+class NodeStuffInWebError extends WebpackError {
+	/**
+	 * @param {DependencyLocation} loc loc
+	 * @param {string} expression expression
+	 * @param {string} shouldEvaluationTo result of evaluation
+	 */
+	constructor(loc, expression, shouldEvaluationTo) {
+		super(
+			`${JSON.stringify(
+				expression
+			)} has been used, it will evaluate to ${shouldEvaluationTo} in next major version`
```

Comment:
There should be more explanation to this.

Something like `global` is a node.js extension to javascript and usually not available in the web.